service_container/autowiring.rst

```diff
@@ -419,20 +459,33 @@ that alias:
         use App\Util\UppercaseTransformer;
         use App\Util\TransformerInterface;
         use App\Service\TwitterClient;
+        use App\Service\MastodonClient;
 
         // ...
         $container->autowire(Rot13Transformer::class);
         $container->autowire(UppercaseTransformer::class);
         $container->setAlias(TransformerInterface::class, Rot13Transformer::class);
+        $container->registerAliasForArgument(
```

Comment:
Using setAlias here might be more clear maybe? registerAliasForArgument would need some specific words for bundle author also.

Reply:
Oh I didn't know I could still use `setAlias` here, I just had a look at the PR from @tgalopin about flysystem

Reply:
So that would be `setAlias(TransformerInterface::class . ' $shoutyTransformer', UppercaseTransformer::class)`, that's not so great is it? Should I put both, for the sake of completeness? Also, I'm not sure what this has to do with bundle authors?

Reply:
Without the spaces yes. That's just more consistent with the other configuration formats for the example.
registerAliasForArgument embeds some logic that is of no use here.
See usage in FrameworkExtension also

Reply:
Ok I replaced all this with `setAlias`